packages/node-cli/src/core/sdksetup/License.js

```diff
@@ -0,0 +1,38 @@
+/*
+ ** Copyright (c) 2022 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the FUTC Oracle License v 1.0 as shown at https://www.oracle.com/downloads/licenses/oracle-free-license.html.
+ */
+'use strict';
+
+const fs = require('fs');
+
+const NodeConsoleLogger = require('../../loggers/NodeConsoleLogger');
+const NodeTranslationService = require('../../services/NodeTranslationService');
+const { COLORS, lineBreak } = require('../../loggers/LoggerConstants');
+const {
+	ERRORS,
+	IMPORTANT,
+	LICENSE_INTRO,
+} = require('../../services/TranslationKeys');
+
+const licenseFileEncoding = 'utf8';
+const licenseFilePath = 'LICENSE.txt';
+
+function display() {
+
+	try {
+		const license = fs.readFileSync(licenseFilePath, licenseFileEncoding);
+		const important = NodeConsoleLogger.formatString(NodeTranslationService.getMessage(IMPORTANT), {
+			color: COLORS.IMPORTANT,
+			bold: true,
+		});
+		const intro = NodeTranslationService.getMessage(LICENSE_INTRO);
+		NodeConsoleLogger.println(`${important}  ${intro}${lineBreak}`, COLORS.IMPORTANT);
+		NodeConsoleLogger.info(license);
+	} catch (err) {
+		NodeConsoleLogger.error(NodeTranslationService.getMessage(ERRORS.MISSING_LICENSE_FILE));
+		process.exit();
```

Comment:
What's the process.exit() here for? Was process suppose to be called here?

Reply:
I think that if we shouldn't be able to display the license, we should display an error and exit, without downloading the jar file. I preferred this quick solution instead of handling rejection, since it's very unlikely this error happens.